src/Service/FilterHelper.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Service;
+
+use Lexik\Bundle\FormFilterBundle\Filter\Condition\ConditionInterface;
+use Lexik\Bundle\FormFilterBundle\Filter\Query\QueryInterface;
+
+class FilterHelper
+{
+    public function applyFilterWithUuids(QueryInterface $filterQuery, $field, $values): ?ConditionInterface
+    {
+        if (empty($values['value'])) {
+            return null;
+        }
+
+        $paramName = sprintf('p_%s', str_replace('.', '_', $field));
+
+        // expression that represent the condition
+        $expression = $filterQuery->getExpr()->eq($field, ':'.$paramName);
+
+        // expression parameters
+        // Added ->getId()->toBinary() to handle Uuid
+
```

Comment:
Remove this blank line to make it more clean that the comment is related to `$parameters = …` and not just some leftover debugging or test code.

Reply:
Removed blank line to avoid confusion.